### ISSUE TITLE
Same menu items differs on Windows and Mac

### ISSUE THREAD

**Issue by aplteam** [Dyalog/ride#945]
On Windows the second menu item in the "Dyalog" menu reads "Preferences".

On the Mac the same menu item reads "Settings".

Whether "Dyalog" is the best name for the menu is questionable as well: under Mac that reads "Ride <version>" which might be better for Windows as well.

**Comment by abrudz** [comment]
I agree that the menu should be renamed RIDE, but the use of Preferences vs Settings is to accommodate common practice on the respective plaftorms.

**Comment by aplteam** [comment]
Which platform? The settings influence both the host and the client, and they might well be different.

Also:

> With macOS Ventura, Apple has revamped the System Preferences app and given it a new name: System Settings. 

Found on https://eshop.macsales.com/blog/84141-how-macos-venturas-system-settings-differs-from-the-previous-system-preferences/

It appears to be more important to be consistent within Dyalog.
